ui/components/playground/PlaygroundContainer.js

```diff
@@ -9,7 +9,7 @@ const settings = {
 
 const PlaygroundContainer = () => (
     <Provider store={store}>
-        <Playground endpoint="http://localhost:8000/graphql" settings={settings} />
+        <Playground endpoint="/graphqldata" settings={settings} />
```

Comment:
[Optional] Ideally this should be done by fetching graphql query (currently it could return some fixed string).